### PR TITLE
armor, loadouts, balancing, guns, oh my

### PR DESCRIPTION
**Pull request by cornercube**
## About The Pull Request
crime fighting with fashion
request by #monolith on discord, as the slim armor vest looks like joshua grahams armor

slim armor vest made into 35 across the board with 0.05 slowdown, to live up to its desc of being slim
original SS13 sec vest is now 40 all-around armor with 0.1 slowdown
OPD armor vest available via loadout, because i dont want to touch deputy balance right now, vest has pockets in it and slight armor buffs and is a smidge weaker than deputy vest

oasis police rifle now re-pathed to 5mm, to make more use of 5mm as a round
and because the police rifle looks more like an assault carbine

big armored vest sprite recycled into the security vest, which will be used to replace the followers slim vest

## Why It's Good For The Game
fashion
balance

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: security vest (combat armor slowdown, really high bullet and melee resist, low laser resist
add: OPD vest, a plain armor vest with pockets
add: armor vest and alt-vest can now spawn at the same rate as combat coat, leather armor replaces armor kit as low spawn
add: more stuff to loadouts
tweak: vest is now 35 resist with 0.05 slowdown, due to being a lightweight vest
tweak: alt armor vest is the old vest, with 40 resist and 0.1 slowdown
balance: bulletproof vest is no longer plateable, as 70 bullet resist is nuts, non-bullet resist a little higher to compensate
balance: armor kits are now only providing token slowdown, to compensate for armor vests being at their level

/:cl:

### CODE REVIEW

**Comment by cornercube** [comment]
> Can you post images please?
ncr

OPD armor vest is identical to the existing slim vest appearance wise

but the big armor vest for the followers will look more like the vault armor they are supposed to be based on

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 1]
```diff
@@ -1105,13 +1102,28 @@
...
 	
+	//Police rifle			Keywords: OASIS, 5mm, Semi-auto, 30 round magazine
+/obj/item/gun/ballistic/automatic/assault_carbine/policerifle
+	name = "Police Rifle"
+	desc = "A pre-war Rifle that has been constantly repaired and rebuilt by the Oasis Police Department. Held together by duct tape and prayers, it somehow still shoots."
+	icon = 'icons/fallout/objects/guns/ballistic.dmi'
+	lefthand_file = 'icons/fallout/onmob/weapons/guns_lefthand.dmi'
+	righthand_file = 'icons/fallout/onmob/weapons/guns_righthand.dmi'
+	icon_state = "rifle-police"
+	extra_damage = 22
```
Comment (OneOneThreeEight) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1113]: psst, i think this deals more damage than the standard assault carbine

Comment (cornercube) [reply code/modules/projectiles/guns/ballistic/automatic.dm:1136]: because its still semi-auto

**Comment by MichaelMaCleod007** [comment]
Hey, the police rifles in the Oasis Armory are still 5.56 and SWAT loadout still has two 5.56 mags instead of two Socom Mags.

**Comment by cornercube** [comment]
> Hey, the police rifles in the Oasis Armory are still 5.56 and SWAT loadout still has two 5.56 mags instead of two Socom Mags.
the ones in the armory are still 5.56 for reasons pertaining to mapping conflicts, so those will be removed once all mapping changes are done

**Pull request merged**